README.md

```diff
@@ -53,7 +60,7 @@ To get started editing the site and seeing your changes, clone this repo and ent
 
 - `cd /path/to/prebid.github.io`
 
-- `bundle install`
+- `bundle install --path vendor/bundle`
```

Comment:
To confirm - should this command be used for everyone going forward?  Or just the mac users who had issues and used the alternate commands mentioned earlier in the page?

Reply:
I think probably everyone - if everyone installs to the local `vendor/bundle` directory there won't be any conflicts with the system's Ruby libs.